src/index.html

```diff
@@ -8,6 +8,54 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+
+    <h1 class="stars stars--0">
```

Comment:
```suggestion
    <div class="stars stars--0">
```
h1-6 tags are tags for headings, consider using divs or uls

Reply:
Thank you! Fixed it